test/components/TruncatedText.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import assert from 'assert';
+import { mount } from 'enzyme';
```

Comment:
Nothing to do now, but for the next time you contribute, we are starting to prefer react-testing-library over enzyme. So we do prefer that is used instead.